lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.spec.ts

```diff
@@ -509,7 +509,7 @@ describe('CardViewTextItemComponent', () => {
             expect(cardViewUpdateService.update).toHaveBeenCalledWith(property, 'updated-value');
         });
 
-        it('should trigger the update event if the editedValue is NOT invalid', async () => {
+        it('should trigger the update event if the editedValue is NOT valid', async () => {
```

Comment:
This doesn't clarify correctly what's going on here. Should the update event be fired when the value is not valid? is this a wanted behavior ? why?